stamina-demo/README.md

```diff
@@ -0,0 +1,161 @@
+# stamina-demo
+
+This project aims to provide a simple demo on how to use stamina to migrate akka-persisted events to their current version. In the demo we will build a simple chat application which logs sent messages starting with a minimalistic version and gradually extending its features. 
+
+## Set up your project to use stamina
+
+The first step would be to depend on the stamina library and configure 
+
+## Step 1 - Simple chat in single chatroom
+
+```scala
+  case class Message(text: String, from: String)
+
+  object ChatRoom {
+    def props = Props(new ChatRoom)
+    val name = "chatroom"
+  }
+
+  class ChatRoom extends PersistentActor {
+    sealed trait Command
+    case class Join extends Command
+    case class SendMessage(msg: Message) extends Command
+
+    sealed trait Event
+    case class MessageSent(msg: Message) extends Event
+    case class Joined(history: Seq[Message]) extends Event
```

Comment:
If `Event` is a 'persistent' event, we should have it extend `Persistable` and `Joined` should probably not extend `Event`